source/indexes/unique-index.txt

```diff
@@ -0,0 +1,59 @@
+.. _pymongo-unique-index:
+
+==============
+Unique Indexes
+==============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+Unique indexes ensure that the indexed fields do not store duplicate values. By
+default, MongoDB creates a unique index on the ``_id`` field during the creation
+of a collection. To create a unique index, specify the field or combination of
+fields that you want to prevent duplication on and set the ``unique`` option to
+``True``.
```

Comment:
S: Might be easier as a list:

```suggestion
of a collection. To create a unique index, perform the following steps:

- Specify the field or combination of fields that you want to prevent duplication on.
- Set the ``unique`` option to``True``.
```